BAZProject/MVVM/Controllers/HomeController/HomeView.swift

```diff
@@ -0,0 +1,150 @@
+        self.setupObservables()
+        self.setupObservablesAction()
+    }
+    
+    func config() {
+        movieTable.register( UINib(nibName: MoviesCell.idReusable , bundle: nil) , forCellReuseIdentifier: MoviesCell.idReusable)
+        movieTable.delegate = self
+        movieTable.dataSource = self
+        searchBar.delegate = self
+        myPicker.delegate = self
+        myPicker.dataSource = self
+        viewModel?.getMovies()
+    }
+    
+    fileprivate func setupObservables() {
+        guard let viewModel = viewModel else { return }
+        viewModel.response.observeNext{ [weak self] response in
+            guard let _ =  response, let self = self else { return }
+            self.movieTable.reloadData()
+        }.dispose(in: bag)
+        
+        viewModel.loading.observeNext{ [weak self] response in
+            guard let self = self else { return }
+            self.loadingView.isHidden = response
+        }.dispose(in: bag)
+        
+        viewModel.error.observeNext{ [weak self] response in
+            guard let response =  response, let self = self else { return }
+            self.alert.alertStyle = .error
+            self.alert.bodyText = response.message
+            self.alert.doneCustomAlertDelegate = self
+            self.present(self.alert , animated: true)
+        }.dispose(in: bag)
+        
+        viewModel.title.observeNext { [weak self] response in
+            guard let response =  response, let self = self else { return }
+            self.navigationItem.title = response
+            self.enablePicker()
+        }.dispose(in: bag)
+    }
+    
+    @IBAction func showPicker(_ sender: Any) {
+        self.enablePicker()
+    }
+    
+    fileprivate func setupObservablesAction() {
+        self.btnCancel.reactive.tap.observeNext{ [weak self] in
+            self?.enablePicker()
+        }.dispose(in: bag)
+        
+        self.btnDone.reactive.tap.observeNext{ [weak self] in
+            self?.viewModel?.getMoviesCategory()
+        }.dispose(in: bag)
+    }
+    
+    func enablePicker(){
+        self.myPicker.isHidden = !self.myPicker.isHidden
+        self.myToolbar.isHidden = !self.myToolbar.isHidden
+    }
+}
+
+extension HomeViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.viewModel?.response.value?.count ?? 0
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = self.viewModel?.response.value else { return UITableViewCell () }
+        if let movieCell = tableView.dequeueReusableCell(withIdentifier: MoviesCell.idReusable, for: indexPath) as? MoviesCell {
+            let item = cell[indexPath.row]
+            movieCell.configure(item, moviesCellDelegate: self)
+            return movieCell
+        } else {
+            return UITableViewCell()
+        }
+    }
```

Comment:
Prefiere usar `guard` en lugar de `if` siempre que sea posible.
Además en este caso es posible usar un solo guard para ambas validaciones.

Además cambiaría el nombre de cell por cellModel o algo más descriptivo. Recuerda que el código debe ser bien legible y entendible. Los nombres que elijas son importantes para eso.
```suggestion
        guard let cellModels = self.viewModel?.response.value, let movieCell = tableView.dequeueReusableCell(withIdentifier: MoviesCell.idReusable, for: indexPath) as? MoviesCell else { return UITableViewCell () }
        let cellModel = cellModels[indexPath.row]
        movieCell.configure(cellModel, moviesCellDelegate: self)
        return movieCell
    }
```